appcenter-data/Data.js

```diff
@@ -56,6 +58,39 @@ function create(documentId, document, partition, writeOptions) {
     return AppCenterReactNativeData.create(documentId, document, partition, writeOptions).then(convertTimestampToDate);
 }
 
+/**
+ * Delete a document.
+ *
+ * @param {string} documentId - The CosmosDB document id.
+ * @param {string} partition - The CosmosDB partition key.
+ * @param {object} writeOptions - Cache write options when the operation is done offline.
+ * @return {Promise} Future asynchronous operation with result being the document with metadata.
+ * If the operation fails, the promise is rejected with an exception containing the details of the error.
+ */
+function remove(documentId, partition, writeOptions) {
+    if (!writeOptions) {
```

Comment:
We use `!writeOptions` here while using `writeOptions === undefined` in other APIs. What is the reason not to check for falsy (`null`, `undefined`) value by `!` in all occurrences?

Reply:
Thanks for catching this. I thought `!writeOptions` would be more appropriate but I forgot to apply the changes to the other APIs. I'll make that change.

Reply:
null is allowed, it's a pass trough.
!writeOptions would allow the API to be called with 0 or false.
Plus we are going to provide typescript definition for this so we should check exact type (undefined).

Reply:
Done